crates/sozo/ops/src/auth.rs

```diff
@@ -198,7 +201,7 @@ pub async fn revoke_writer<A>(
     new_writers: &[ResourceWriter],
     txn_config: &TxnConfig,
     default_namespace: &str,
-    walnut_debugger: &Option<WalnutDebugger>,
+    #[cfg(feature = "walnut")] walnut_debugger: &Option<WalnutDebugger>,
```

Comment:
weird why fmt is using only one line in this case.

Reply:
yeah im not sure, i dont like the look of it as well but oh well